### DISCUSSION TITLE
第三方组件不显示，怎么回事？

### DISCUSSION THREAD

**Discussion by elarbor** [storybookjs/storybook#29327] [Help]
原本的项目引入的第三方 ant-design 组件库。在项目里是正常的，在使用文档的预览出不来

![image](https://github.com/user-attachments/assets/c3327542-bcb4-4e6d-97fa-ed8f6463eb29)

![image](https://github.com/user-attachments/assets/30cfcfe7-de38-4645-9980-14a5a50f9b51)

**Comment by valentinpalkovic** [comment]
Please formulate your question in English and open a new discussion. Thanks!
